tests/core/test_apd.py

```diff
@@ -702,37 +739,17 @@ def test_parse_date_field_00(input_, expected):
 
 
 @pytest.mark.parametrize('input_,expected', (
-    (
-        '>Deceased: </strong> Luis Fernando Martinez-Vertiz | Hispanic male | 04/03/1994</p>',
-        'Luis Fernando Martinez-Vertiz | Hispanic male | 04/03/1994',
-    ),
-    (
-        '>Deceased: </strong> Cecil Wade Walker, White male, D.O.B. 3-7-70<',
-        'Cecil Wade Walker, White male, D.O.B. 3-7-70',
-    ),
-    (
-        '>Deceased: </span></strong> Halbert Glen Hendricks - Black male - 9-24-78<',
-        'Halbert Glen Hendricks - Black male - 9-24-78',
-    ),
-    (
-        '',
-        '',
-    ),
-    (
-        '<strong>Deceased:  </strong>Hispanic male, 19 years of age<br>',
-        'Hispanic male, 19 years of age',
-    ),
-    pytest.param(
-        '<p>	<strong>Deceased:   </strong>David John Medrano,<strong> </strong>Hispanic male, D.O.B. 6-9-70</p>',
-        'David John Medrano, Hispanic male, D.O.B. 6-9-70',
-        marks=pytest.mark.xfail(
-            reason='Do not know how to parse this one as the "<strong> </strong>" is throwing the regex off.'),
-    ),
+    ('traffic-fatality-2-3', 'White female, DOB 02/15/1960'),
+    ('traffic-fatality-4-6', 'White female, DOB 12/31/1960'),
+    ('traffic-fatality-20-4', 'Hispanic male, 19 years of age'),
+    ('traffic-fatality-25-4', ', Hispanic male, D.O.B. 6-9-70'),
+    ('traffic-fatality-73-2', 'White male, DOB 02/09/80'),
```

Comment:
The old style of tests weren't really adequate though. Remember we had issue #92 (Deceased fields that can't be parsed because they contain no DOB), and I got the old-style test passing for https://austintexas.gov/news/traffic-fatality-20-4 in #125. But then you had to open #150, because the same bulletin still wasn't being scraped correctly, so I added these tests that check the parsing of the whole page. Also, I think BeautifulSoup depends on having a valid HTML tree to select text from.